### ISSUE TITLE
Utilise validation and testing datasets from Open Images

### ISSUE THREAD

**Issue by ferrouswheel** [singnet/open-images-annotation#4]
Currently only the training data is used from Open Images and this is partitioned into a training/test set when exported to the dlib xml format.

This requires:
- Make AnnotationSets understand validation/test data
- Make data loading from OpenImages use the alternative data locations of validation/test.
- Make the dlib xml converter use validation and test data.
